### ISSUE TITLE
The Application does not work with Kubernetes Nodes when deployed with NodePort Mode service

### ISSUE THREAD

**Issue by ubaidsworld** [iam-veeramalla/go-web-app-devops#11]
Hi Abhishek,
I have been trying to reach out to you regarding my failed attempts on Kubernetes. I have tried implementing the process using your video. But I am not able to run the application through NodePort Mode and Kubernetes Node EXTERNAL-IP. Please let me know if there is any setting I am missing.
I have got no other medium to talk to you so I am using this issue section to let you know the problem I am facing while trying to deploy your application.
Please let me know what is your thought process. I am not able to ping the node EXTERNAL-IP through browser or curl command.
Please your guidance is required.
I am looking for forward to hearing from you soon.

**Comment by ManojKumarN001** [comment]
@ubaidsworld same here too facing same issue

**Comment by ubaidsworld** [comment]
@ManojKumarN001  Hi I have resolve this problem. Go to Security Groups and change the Port access of Custom TCP to All TCP and it will work. It is not best practice thought but it will help you see you application. Try it.

**Comment by ManojKumarN001** [comment]
@ubaidsworld Yeah tried it will work 
but how abhishek is doing without modifying the SG

**Comment by ubaidsworld** [comment]
@ManojKumarN001 I wonder the same thing. There were possibility that author wants us to discover that thing. Or may be wants us to subscribe his channel so he can earn from what he is teaching. I had tried contacting him but there is no response from his side. So raising issue was the last attempt I made, thinking that the author might respond. But he turn off deaf ears on the issues people raises. So my doubt became a little strong about my second thought over subscription.
But I am glad that you got it resolved from my suggestion.
